src/index.tsx

```diff
@@ -4,12 +4,7 @@ import './index.css';
 import { App } from './App';
 import reportWebVitals from './reportWebVitals';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root'),
-);
+ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
With `strict mode` an app works but there is an error in console according to `findDOMNode is depracted in Strict Mode.` - it's related to `Accordion` component. More info you can find on these issues: https://github.com/mui-org/material-ui/issues/20561 & https://github.com/mui-org/material-ui/issues/13394 . I didn't find another solution for this error except disabling `Strict mode`.
![strict-mode](https://user-images.githubusercontent.com/29304012/115905713-c5f94800-a466-11eb-94f3-6f2485c4b274.png)

Reply:
I see, that's not ideal :/ I guess there's little we can do. An alternative is to use unstable theme creator, but it's not recommended in production. TBH, I'm wondering if we should not just leave StrictMode on and get used to this warning - what do you think? 
I'll try to figure out a way of passing a ref to our accordion, but it could be problematic, because the error is thrown by Transition - an element that Accordion is using under the hood.

Reply:
I'm afraid that if we leave this `console.error` warning it can be misleading for us as we (or next devs) work in next weeks/months why this error persists.

Reply:
The good news is that Material v. 5 will resolve this problem. I'll add a comment that StrictMode should be restored when v. 5 is stable :)